promql/functions.go

```diff
@@ -32,29 +33,42 @@ type Function struct {
 	ArgTypes   []ValueType
 	Variadic   int
 	ReturnType ValueType
-	Call       func(ev *evaluator, args Expressions) Value
+
+	// vals is a list of the evaluated arguments for the function call.
+	//    For range vectors it will be a Matrix with one series, instant vectors a
+	//    Vector, scalars a Vector with one series whose value is the scalar
+	//    value,and nil for strings.
+	// args are the original arguments to the function, where you can access
+	//    matrixSelectors, vectorSelectors, and StringLiterals.
+	// ts is the evaluation timestamp.
+	// enh.out is a pre-allocated empty vector that you may use to accumulate
+	//    output before returning it. The vectors in vals should not be returned.a
+	// Range vector functions need only return a vector with the right value,
+	//     the metric and timestamp are not neded.
+	// Range vector functions need only return a vector with the right value and
```

Comment:
There seems to be some duplication here. Or incomplete copy-pasta.

`s/Range vector/Vector/` ?